src/StdDeQueue.cc.in

```diff
@@ -0,0 +1,8 @@
+#include "app-framework/Queues/StdDeQueue.hh"
+#include "app-framework-base/DataTypes.hh"
+
+namespace appframework {
+   typedef StdDeQueue<@MAKE_DATATYPE_CPPTYPE@> @MAKE_DATATYPE_PREFIX@StdDeQueue;
```

Comment:
Ah, I guess this is the problem. I would solve it removing the allocation via CETlib for queues. It's pointless anyway if we have a switch for the allocation.